pkg/utils/utils.go

```diff
@@ -93,9 +100,28 @@ func ExecuteCommand(command string, timeout time.Duration, context *interactive.
 	test, err := tnf.NewTest(context.GetExpecter(), *tester, handlers, context.GetErrorChannel())
 	gomega.Expect(err).To(gomega.BeNil())
 	gomega.Expect(tester).ToNot(gomega.BeNil())
-
-	test.RunAndValidateWithFailureCallback(failureCallbackFun)
-
+	testok := false
+	if runopt == "" {
+		testok = true
+		test.RunAndValidateWithFailureCallback(failureCallbackFun)
+	} else {
+		var message string
+		test.RunWithCallbacks(func() {
+			testok = true
+			message = sucssesmsg
+		}, func() {
+			message = failmsg
+		}, func(e error) {
+			message = errmsg
+		})
+		_, err = ginkgo.GinkgoWriter.Write([]byte(message))
```

Comment:
You may want to use tnf.ClaimFilePrintf(), which is a helper that does GinkgoWriter.Write() with associated error handling for you.